docs/man/xrdp-sesrun.8.in

```diff
@@ -4,7 +4,7 @@ xrdp\-sesrun \- \fBsesman\fR(8) session launcher
 
 .SH "SYNTAX"
 .B xrdp\-sesrun
-.I server username password width height bpp
+.I server username password width height bpp type
```

Comment:
This quick-fix looks good to me but I have thought about this long ago.
In the long term, it would be nice to specify the session type by name rather than code.

Reply:
Good point. I'll revise the entire command line interface I think, so it's a bit more sensible.